canonical-websites/jaas.ai
# Add charm details page

## Done
Added the template and data for the charm details page.

## QA
- Go to /wordpress
- See that there is content
- Check that /haproxy/42 loads version 42

## Details
Fixes https://github.com/ubuntudesign/juju-squad/issues/348
